src/Artsy/Router/RouterLink.tsx

```diff
@@ -41,8 +42,22 @@ export const RouterLink: React.FC<LinkProps> = ({ to, children, ...props }) => {
       ...handlers,
     ])
 
+    // Wrap any provided onClick with pageview tracking
+    // if the page type stays the same. When linking across apps
+    // those components will get mounted with their own tracking.
+    // But when linking within the same app (just different params),
+    // we want to trigger a pageview.
+    const onClick = e => {
```

Comment:
So..yea. Finding that when using this to navigate from a collection to another collection, we want to trigger the pageview.

But, when navigating from an artist page to a collection, we'd want to defer to the regular tracking happening on mount in those apps.

This so far is working (in the current production setting of just having client side routing in the collect suite of apps), as well as with the flag for global client side routing turned on (subsequent pageviews in the same type _are_ triggered).